src/pages/management/configuration-api/v3.6/index.mdx

```diff
@@ -1195,6 +1194,72 @@ Unlike Agents, Bot don't have passwords or emails - you cannot log in as a Bot.
+
+#### Request
+
+| Parameter                                | Required | Data type  | Notes                                                                                                                                                                                                                                                                                              |
+| ---------------------------------------- | -------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`                                   | Yes      | `string`   | Display name                                                                                                                                                                                                                                                                                       |
+| `avatar`                                 | No       | `string`   | Avatar URL                                                                                                                                                                                                                                                                                         |
+| `max_chats_count`                        | No       | `int`      | Max. number of incoming chats that can be routed to the Bot; default: 6. Limited to 500.                                                                                                                                                                                                           |
+| `default_group_priority`<sup>**1**</sup> | No       | `string`   | The default routing priority for a group without defined priority.                                                                                                                                                                                                                                 |
+| `job_title`                              | No       | `string`   | Bot's job title                                                                                                                                                                                                                                                                                    |
+| `owner_client_id`                        | Yes      | `string`   | Required only when authorizing via [PATs](/authorization/authorizing-api-calls/#personal-access-tokens). When you provide this param while authorizing with a Bearer Token, the `client_id` associated with the Bearer Token will be ignored, and provided `owner_client_id` will be used instead. |
```

Comment:
check the link. Perhaps it should point to /authorization/agent-authorization#personal-access-tokens